test/ctor-eval/strings.wast

```diff
@@ -0,0 +1,16 @@
+(module
```

Comment:
I don't suppose you would want to start a `test/lit/ctor-eval` directory and make this a lit test, would you?

Reply:
I think we discussed this last time as well, and there was some technical issue iirc? Each of these tests is spread over 3 files now, that might be it.

Anyhow, if you're not happy with adding more here, I can make the PR do a little more in that area, enough for Precompute to be used for testing instead. I pushed that now.